### ISSUE TITLE
2.4 build error against PostgreSQL < 15

### ISSUE THREAD

**Issue by devrimgunduz** [MigOpsRepos/credcheck#29]
Hi,

2.4 fails to build against PostgreSQL < 15. Is that intentional? I could not see anything about that in the release notes.

Thanks! Devrim

```
/usr/bin/clang -Wno-ignored-attributes -fno-strict-aliasing -fwrapv -fexcess-precision=standard -Wno-unused-command-line-argument -Wno-compound-token-split-by-macro -Wno-deprecated-non-prototype -O2  -Wno-ignored-attributes -I. -I./ -I/usr/pgsql-14/include/server -I/usr/pgsql-14/include/internal  -D_GNU_SOURCE -I/usr/include/libxml2  -I/usr/include -flto=thin -emit-llvm -c -o credcheck.bc credcheck.c
credcheck.c:1331:2: error: call to undeclared function 'MarkGUCPrefixReserved'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
 1331 |         MarkGUCPrefixReserved("credcheck");
      |         ^
1 error generated.

```

**Comment by darold** [comment]
hum, right I have missed this check. On the way to be fixed.

**Comment by devrimgunduz** [comment]
Can you please release as 2.5 or so? 2.4 was out for an hour, and it may break some automation stuff.

**Comment by darold** [comment]
Hum, I was thinking this was shorter but you are right, I will get out a 2.5 release asap.

**Comment by darold** [comment]
Done.

**Comment by devrimgunduz** [comment]
Thanks!

**Comment by devrimgunduz** [comment]
Oops:

2.5 is badly broken because of this commit:

https://github.com/MigOpsRepos/credcheck/commit/975632af05322a577cb3a9ef34066641e16c07af#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52

Output:

make[1]: /usr/lib/postgresql/15/bin/pg_config: No such file or directory

**Comment by darold** [comment]
Arrg I forgot to remove the full path for the test.

**Comment by darold** [comment]
That's not my day, there was also an issue in the upgrade file. I have fixed the v2.5.0 release but also generated a new one v2.6.0.

**Comment by darold** [comment]
@devrimgunduz my apologies for the extra work.
